lib/collections/users/collection.js

```diff
@@ -1,34 +1,73 @@
 Meteor.users.helpers({
-  fullName() {
-    return `${this.profile.firstName} ${this.profile.lastName}`;
-  },
-
-  email() {
-    return this.emails[0].address || this.profile.publicEmail;
-  },
-
-  profilePicture() {
-    var image = Images.findOne({ '_id': this.profile.image });
-
-    if (image)
-      return image;
-    else {
-      var defaultPictureIndex = UserUtils.getDefaultPictureIndex(this._id);
-      return { url: `/images/default_${defaultPictureIndex}.png` };
-    }
+    fullName() {
+        return `${this.profile.firstName} ${this.profile.lastName}`;
+    },
+
+    email() {
+        return this.emails[0].address || this.profile.publicEmail;
+    },
+
+    profilePicture() {
+        var image = Images.findOne({
+            '_id': this.profile.image
+        });
+
+        if (image)
+            return image;
+        else {
+            var defaultPictureIndex = UserUtils.getDefaultPictureIndex(this._id);
+            return {
+                url: `/images/default_${defaultPictureIndex}.png`
+            };
+        }
 
-  },
+    },
+    allAnswersCount(userId) {
+        var answersCount = 0;
+        var bestAnswersCount = 0;
+        var questions = Questions.find({}).fetch();
+        questions.forEach(function(question) {
+            var answers = question.answers
+            for (var i = 0; i < answers.length; i++) {
+                var answer = Answers.find({
+                    _id: answers[i]
+                }).fetch();
+                if (answer[0].ownerId == userId) {
```

Comment:
- in that case you can 
  - use findOne instead of find .. which will return only one document.
  - add `[0]` after the fetch directly since the name `answer` implies that it's not an array.
- also you don't have to loop all questions to get their answers .. since you are doing this to all questions. you can just fetch all answers documents. which will simplify this function to 
```js
var answersCount = Answers.find({ownerId: userId).count;
var bestAnswersCount =  Answers.find({ownerId: userId, bestAnswer:true).count;
```
you just have to make sure that you delete questions answers when deleting it. which we already do [here](https://github.com/desoukya/SEMS/blob/d8100277449266679071e0130d7034312dbc90dc/server/methods/questions.js#L62)